cmd/hil-vpn-privop/openvpn-conf.go

```diff
@@ -80,6 +82,28 @@ func (cfg OpenVpnCfg) Save() error {
 	return err
 }
 
+// Return a cryptographically-random 12-character base64(url) encoded string.
+// This is to do collision avoidance given the 15-character limit on network
+// interface names. We still prefix interface names with tap for a modicum
+// of readability. See also issue #14.
```

Comment:
We actually *need* to prefix interface names with `tap` if we don't explicitly specify `dev-type tap` in our openvpn configuration file (which we aren't). 
https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/

Reply:
Good catch. I updated the comment to mention this.